DoSomething/gladiator
# User activity

#### What's this PR do?

Show's user activity in the  `users.show` view for each competition that a user is in, and adds user activity to the competition CSV. 

NOTE: #57 asked for this to be shown in the competition view, but we decided for this first step, before any leaderboard behavior, we would show it on the user view.

#### How should this be manually tested?
Pull down, view a user that is in a competition, and you should see there reportback information, if they have submitted a reportback.

#### Any background context you want to provide?

Currently this looks like: 

<img width="918" alt="screen shot 2016-03-21 at 4 18 38 am" src="https://cloud.githubusercontent.com/assets/1700409/13921161/efd48600-ef4c-11e5-8fd0-c1f50211fd9a.png">

But we can certainly update this to show more information and/or show a more detailed version of the reportback activity in another route. This just shows the info that was asked of in the issues. 

#### What are the relevant tickets?
Fixes #57 and Fixes #108 

@weerd @angaither @deadlybutter
